examples/simple-console/src/main/java/com/snowplowanalytics/Main.java

```diff
@@ -52,7 +45,7 @@ public static void main(String[] args) {
         // build an emitter, this is used by the tracker to batch and schedule transmission of events
         BatchEmitter emitter = BatchEmitter.builder()
                 .url(collectorEndpoint)
-                .bufferSize(4) // send an event every time one is given (no batching). In production this number should be higher, depending on the size/event volume
+                .bufferSize(4) // send batches of 4 events. In production this number should be higher, depending on the size/event volume
```

Comment:
I don't think the batching would be executed because we force the tracker to close as soon as the events have been submitted to the tracker. Closing the tracker the flushBuffer would send all of them in a single shot whatever is the size of the buffer.
I don't suggest to change anything for this specific issue, it's more a consideration for future improvements.

Reply:
ops... noticed now the discussion above on the same topic 🤦 :)

Reply:
While we're talking about future improvements, I note that using `mavenLocal()` is [discouraged](https://medium.com/decisionbrain/when-to-not-use-mavenlocal-in-your-gradle-build-script-6da03902f9df)! Probably not a problem since this is a little demo but could be worth considering next time it's updated.